src/sentry/runner/__init__.py

```diff
@@ -161,4 +163,26 @@ def call_command(name, obj=None, **kwargs):
 
 
 def main():
-    cli(prog_name=get_prog(), obj={}, max_content_width=100)
+    # This variable is *only* set as part of direnv/.envrc, thus, we cannot affect production
+    if os.environ.get("SENTRY_DEVENV_DSN"):
+        # We do this here because `configure_structlog` executes later
+        logging.basicConfig(format="%(levelname)s:%(message)s", level=logging.INFO)
+        logger = logging.getLogger(__name__)
+
+        logger.info(
+            "The Sentry runner will report development issues to Sentry.io. "
+            "Use SENTRY_DEVENV_NO_REPORT to avoid reporting issues."
+        )
+        try:
+            cli(prog_name=get_prog(), obj={}, max_content_width=100)
```

Comment:
In the previous commit, I only had `cli()` once, however, I had 2-3 if conditions. This makes it easier to read but will require the next person to ever touch the command to also update it below.

My apologies for the multiple comments (and probably multiple emails). I was only expecting to make 1 single comment.

Reply:
Should we refactor this to a new function so that we don't need to update both `cli()` call sites?

Reply:
@billyvg I've used this:

```diff
diff --git a/src/sentry/runner/__init__.py b/src/sentry/runner/__init__.py
index 6522718a7d..436c5b1e0a 100644
--- a/src/sentry/runner/__init__.py
+++ b/src/sentry/runner/__init__.py
@@ -163,6 +163,12 @@ def call_command(name, obj=None, **kwargs):


 def main():
+    func = cli
+    kwargs = {
+        "prog_name": get_prog(),
+        "obj": {},
+        "max_content_width": 100,
+    }
     # This variable is *only* set as part of direnv/.envrc, thus, we cannot affect production
     if os.environ.get("SENTRY_DEVENV_DSN"):
         # We do this here because `configure_structlog` executes later
@@ -174,7 +180,7 @@ def main():
             "Use SENTRY_DEVENV_NO_REPORT to avoid reporting issues."
         )
         try:
-            cli(prog_name=get_prog(), obj={}, max_content_width=100)
+            func(**kwargs)
         except Exception as e:
             # This reports to the project sentry-dev-env
             with sentry_sdk.init(dsn=os.environ["SENTRY_DEVENV_DSN"]):
@@ -184,5 +190,4 @@ def main():
                 logger.info("We have reported the error below to Sentry")
             raise e
     else:
-        # If you change this command here make sure to also change it in the block above
-        cli(prog_name=get_prog(), obj={}, max_content_width=100)
+        func(**kwargs)
```